customisation/crawling.md

```diff
@@ -1,6 +1,27 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Crawling
 description: Crawling the web for machine translation training data
 ---
+
+**Crawling** or **web scraping** is the process of automatically collecting large amounts of [training data](/customisation/training-data.md) for [language models](/concepts/language-model.md) and machine translation systems.
+Web crawling is used to acquire [parallel data](/customisation/parallel-data.md) by visiting bilingual or multilingual websites and extracting parallel text sets.
+Another approach is to use crawling to collect monolingual text in a target language.
+
+### Process
+
+1. Visiting the first URL, manually specified by the user or selected from a pre-defined list of URLs
+2. Extracting URLs from the visited web page and adding them to a queue of URLs to be visited
+3. Extracting text data from the visited web page and storing it in a database for later use
+4. URL filtering to determine which URLs to visit next based on the pre-defined criteria
+5. Repetition of URL extraction, text extraction, and URL filtering for each URL in the queue
+6. Preprocessing and [filtering](/customisation/filtering.md) of the extracted text data to remove noise and normalise quality
```

Comment:
Let's use the infinitive for processes, as we did (you did, haha) in the [Attention](https://machinetranslate.org/attention#process) article.